licensed_pile/write.py

```diff
@@ -120,9 +120,9 @@ def process_single(
                 update_interval = kwargs.pop("update_interval", 1)
                 debug = kwargs.pop("debug", False)
 
-                for i, line in enumerate(f):
-                    with logger(line=i):
-                        try:
+                try:
```

Comment:
This catches exceptions that happen when reading lines from the file? I guess due to file corruption or something?